src/consts/models.tsx

```diff
@@ -116,3 +127,109 @@ export interface Filters {
   toAmount?: number;
   transactionType?: string;
 }
+
+export type MainTabNavigatorParamList = {
+  [Route.Dashboard]: undefined;
+  [Route.ContactList]: undefined;
+  [Route.Settings]: undefined;
+};
+
+export type RootStackParamList = {
```

Comment:
technically it's not a list 😀

Reply:
Yes but I wanted to keep it consistent with official documentation